src/Spice86/ViewModels/BreakpointViewModel.cs

```diff
@@ -54,7 +80,7 @@ public bool IsEnabled {
 
     public bool IsRemovedOnTrigger { get; }
 
-    public uint Address { get; }
+    public long Address { get; }
```

Comment:
This is a breaking change I did not expect. Do we have a usecase for addresses below 0 or above 0xFFFFFFFF ?

Reply:
It represent a trigger : an I/O port number, a cycles number, a memory address, or an interrupt number.

Guess it could be int.